### ISSUE TITLE
Feature Request - Run command at startup

### ISSUE THREAD

**Issue by PhilJbt** [xpipe-io/xpipe#113]
It would be very useful to be able to define a command line to be executed as soon as connected to a remote device.
This should be stored crypted.

**Comment by crschnick** [comment]
You can already do that.

Either using a shell environment if you want to run it on one system or using a script if you want to run it on every remote system. For a script, you have to click on the dropdown and then enable the run on init toggle.

About the crypted storage, can you elaborate a bit on why these command should be treated as secret information? Even when they are stored encrypted, executed commands are shown in logs and more, so I don't think this would make much sense.

**Comment by PhilJbt** [comment]
My bad, I didn't understand well what `Shell Env` was used for, this feature answers to my needs.
About encryption, it's because I implemented a passphrases system in my own tools, which are hosted on a dedicated server. I don't want my tools passphrases to be stored unencrypted anywhere.

Also, I had to recreate the connection entry since it disappeared after close Xpipe, I doubt that's the expected behavior, even on the portable build, isn't it?
